01_01_Basic_Rules.txt

```diff
@@ -668,42 +668,43 @@ RP / Using Skills
-take down an opponent, the GM can either have you roll a stealth check and an 
-unarmed combat check, or double the DC, and have you roll once adding both 
-modifiers. Also add any relevant Stat Bonuses to a roll. When rolling 
+apply any and all relevant skills to your result. Also add any relevant Stat Bonuses to a roll. When rolling 
 awareness, add your PER Bonus, etc.
 
 	At any time out of combat, you may decide to use one of your skills. If you 
-are not pressured by imminent / immediate danger, you may "take a 100%" to pass 
-a skill check, then apply any ranks in the skill. When rushed by a time 
-constraint or danger, you can "take a 50%" to pass a skill check, then apply any 
-ranks in said skill. However, some checks cannot be passed in this way. If you 
-are staring at a ship''s blast doors, and want to kick them down, you can't take
-additional time to increase your odds of success.
+are not pressured by imminent / immediate danger, you may "take a 100%" to take your
+time with a task to roll exactly 100 on a skill check, then apply any ranks in the
+skill. When rushed by a time constraint or danger, you can "take a 50%" to take
+slightly more time than a 1-action, in-combat check on a task to roll exactly 50 on
+a skill check, then apply any ranks in said skill. However, some checks cannot be
+passed in this way. If you are staring at a ship's blast doors, and want to kick
+them down, you can't take additional time to increase your odds of success.
 
 	If you want to use a skill in combat, see the "Combat: Actions" subsection.
 Skill ranks are exact percentiles to pass a skill check. If you have 50 ranks 
 in a skill, then you have a +50% chance to pass a skill check for that skill. 
-Every time you purchase a new skill for 10 Skill points, it starts as a +10% 
-skill. When leveling up you may add up to 10 points to any one skill.
+When leveling up you may add up to 5 points to any skill.
 
-	Normally a character can attempt any skill check whether or not they have 
-invested in learning the skill; however, sometimes a character will have more 
-penalties if they attempt to use a skill that they don't have "Proficiency" in. 
-A character has Proficiency in a skill when they have 10 or more skill points in 
-that skill.
+	In many cases, a character can attempt a skill check whether or not they
+have invested in learning the skill; however, your GM may decide otherwise, apply
+penalties to your roll, raise the DC, ask you to justify why your character can
+attempt the roll, or make any other judgement they deem appropriate. 
 
-	Defenders win ties for opposing rolls. If a character ties a DC they fail.
+	Characters are considered to be Proficient with certain skills, based on
+their class and background. When a character is Proficient in a skill, each skill 
+point they spend in that skill upon leveling up increases that skill by two 
+instead of one.
+
+	Defenders win ties for opposing rolls.

```

Comment:
A small concern that this is inconsistent with attackers succeeding on ties with the accuracy DC.

Reply:
You are correct X|
https://github.com/bleehu/Compound_X/blob/dev/01_01_Basic_Rules.txt#L1031